### PR TITLE
Update 2.1 to March Release

### PR DESCRIPTION
**Pull request by adaggarwal**

### CODE REVIEW

**Comment by adaggarwal** [comment]
Intermittent #1449 failures for osx

**Comment by adaggarwal** [comment]
Ready for review.

**Comment by adaggarwal** [comment]
Ready for review.

File: patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch [hunk 1]
```diff
@@ -0,0 +1,31 @@
+From f2a4abfda4f2dea0d1bece590222b304b6242cf3 Mon Sep 17 00:00:00 2001
+From: adaggarwal <aditya.aggarwal@microsoft.com>
+Date: Fri, 6 Mar 2020 18:25:45 +0000
+Subject: [PATCH] glibc sysctl deprecation handling
```
Comment (omajid) [inline patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch]: I proposed this in coreclr and they suggested an alternative fix. That discussion is here: https://github.com/dotnet/coreclr/pull/28010.  This is the final fix merged into release/3.1: https://github.com/dotnet/coreclr/pull/28012

Comment (adaggarwal) [reply patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch]: Thanks for the pointers.

Comment (adaggarwal) [reply patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch]: We'll make sure to add the warning work-around patch if we don't get updated coreclr with the fix in next release.

Comment (omajid) [reply patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch]: I am not sure if I got that fix into 2.1. Should I look into getting it in there too?

Comment (adaggarwal) [reply patches/coreclr/0001-glibc-sysctl-deprecation-handling.patch]: Yep. That would help in getting rid of the patch next release.

**Pull request merged**